src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -629,36 +644,39 @@ def _visit_map(self, node: gtir.FunCall) -> ValueExpr:
         assert isinstance(node.type.element_type, ts.ScalarType)
         dc_dtype = dace_utils.as_dace_type(node.type.element_type)
 
-        input_args = [self.visit(arg) for arg in node.args]
-        input_connectors = [f"__arg{i}" for i in range(len(input_args))]
+        input_connectors = [f"__arg{i}" for i in range(len(node.args))]
         output_connector = "__out"
 
         # Here we build the body of the tasklet
         fun_node = im.call(node.fun.args[0])(*input_connectors)
         fun_python_code = gtir_python_codegen.get_source(fun_node)
         tasklet_expression = f"{output_connector} = {fun_python_code}"
 
-        input_local_offsets = [
-            input_arg.local_offset for input_arg in input_args if input_arg.local_offset is not None
-        ]
-        if len(input_local_offsets) == 0:
+        input_args = [self.visit(arg) for arg in node.args]
+        input_connectivities: dict[gtx_common.Dimension, gtx_common.Connectivity] = {}
+        for input_arg in input_args:
+            assert isinstance(input_arg.gt_dtype, itir_ts.ListType)
+            assert input_arg.gt_dtype.offset_type is not None
+            offset_type = input_arg.gt_dtype.offset_type
+            if offset_type == _CONST_DIM:
+                # this input argument is the result of `make_const_list`
+                continue
+            offset_provider = self.subgraph_builder.get_offset_provider(offset_type.value)
+            assert isinstance(offset_provider, gtx_common.Connectivity)
+            input_connectivities[offset_type] = offset_provider
+
+        if len(input_connectivities) == 0:
```

Comment:
As we discussed, some icon4py stencils currently have different `offset_type` for the input to `map_`, and therefore `len(input_connectivities) != 1`, as below:
```
model/atmosphere/dycore/tests/dycore_stencil_tests/test_compute_contravariant_correction_of_w.py 

as_fieldop(λ(__arg0__, __arg1__) → map_(multiplies)(·__arg0__, ·__arg1__),
           u⟨ Cellₕ: [horizontal_start, horizontal_end), Kᵥ: [vertical_start, vertical_end) ⟩)(
  as_fieldop(λ(it_) → neighbors(C2CEₒ, it_),
             u⟨ Cellₕ: [horizontal_start, horizontal_end), Kᵥ: [vertical_start, vertical_end) ⟩)(
    e_bln_c_s
  ), as_fieldop(λ(it_) → neighbors(C2Eₒ, it_),
                u⟨ Cellₕ: [horizontal_start, horizontal_end), Kᵥ: [vertical_start, vertical_end) ⟩)(
       as_fieldop(λ(__val) → ·__val, u⟨ Edgeₕ: [0, 27), Kᵥ: [vertical_start, vertical_end) ⟩)(
         z_w_concorr_me
       )
     )
)
```

Reply:
but both should have same `has_skip_value` property. If not, I would consider it a bug in icon4py.